src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -539,19 +540,24 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
     }
 
     renderUpgradeLinkDescription() {
-        const { trackingProps = {} } = this.props;
+        const { openUpgradePlanModal, trackingProps = {} } = this.props;
         const { inviteCollabsEmailTracking = {} } = trackingProps;
         const { upgradeLinkProps = {} } = inviteCollabsEmailTracking;
 
         return (
             <div className="upgrade-description">
-                <UpgradeBadge type="warning" />
+                <UpgradeBadge />
                 <FormattedMessage
                     values={{
                         upgradeGetMoreAccessControlsLink: (
-                            <Link className="upgrade-link" href="/upgrade" {...upgradeLinkProps}>
+                            <PlainButton
+                                className="upgrade-link"
+                                onClick={openUpgradePlanModal}
```

Comment:
Should we make this backwards compatible? openUpgradePlanModal isn't passed in as a prop yet right? 

Other than this, everything else looks good Jackie!

Reply:
good call, I will default it to null up top